.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+
+language: node_js
+notifications:
+  email: false
+node_js:
+- stable
+cache: npm
+stages:
+  - test
+  - name: deploy
+    # Don't deploy on tags or forked repositories
+    if: type = push AND fork = false
+jobs:
+  include:
+    ### START TEST STAGE ###
+    - stage: test
+      script: npm run lint && npm run test
+    ### END TEST STAGE ###
+    ### START DEPLOY STAGE ###
+    - stage: deploy
+      name: "Deploy to now.sh"
+      if: branch = develop OR branch = staging OR branch = master
```

Comment:
We aren't using a staging branch but I guess this wont cause any issues.

Reply:
I don't know if you are going to use the same branching model we have discussed for the app-backend (feature branches from dev, dev to staging branch for pre-releases and then staging to master for final release).

If you decide to merge dev to master directly I will remove the staging branch from the condition.

Reply:
I've just removed the staging branch since it seems that nobody is going to use it. ANd even if - we can just reenable it without much hassle.